test/simplifyExpression/simplify.test.js

```diff
@@ -163,6 +163,14 @@ describe('handles unnecessary parens at root level', function() {
   tests.forEach(t => testSimplify(t[0], t[1], t[2]));
 });
 
+describe('multiplication of powers support', function(){
```

Comment:
Actually I don't think you need these tests, since this would be the answer it'd get to anyways even if it didn't have multiplication of powers support - it'd just get to the answer a different way. And having the tests here might be confusing then